tqecjs/src/workspace/components/notifier.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @fileoverview This file contains the function that shows a notification on the screen.
+ * The notification is used to display messages to the user.
+ */
+
+import { Graphics, Text, Container } from 'pixi.js';
+
+/**
+ *
+ * @param {*} text'
+ * @description This function shows a notification on the screen.
+ */
+export default function notification(app, text) {
+	// Create a container for the notification
+	const notificationContainer = new Container();
+
+	// Create a text object for the notification text
+	const notificationText = new Text(text, {
+		fontSize: 15,
+		fill: 0xffffff,
+		fontWeight: 'bold',
+	});
+	notificationText.anchor.set(0.5);
```

Comment:
Will anchors be preferred over pivots in this project?

Reply:
it's just a notification text so I wouldn't think too hard at the moment.

Reply:
we can discuss this once other aspects of the code base stabilizes :)